wandb/sdk/artifacts/artifact.py

```diff
@@ -70,9 +70,9 @@
 from wandb.sdk.lib import filesystem, retry, runid, telemetry
 from wandb.sdk.lib.deprecate import Deprecated, deprecate
 from wandb.sdk.lib.hashutil import B64MD5, b64_to_hex_id, md5_file_b64
-from wandb.sdk.lib.mailbox import Mailbox
 from wandb.sdk.lib.paths import FilePathStr, LogicalPath, StrPath, URIStr
 from wandb.sdk.lib.runid import generate_id
+from wandb.sdk.mailbox import Mailbox
```

Comment:
you don't think it should be under lib?

Reply:
i think the second point is a valid case, original we were putting everything that a "library" or utility in lib, but it's true that mailbox is more like interface so it valid to have them both on the same level